rastervision_pytorch_learner/rastervision/pytorch_learner/learner_config.py

```diff
@@ -150,6 +150,8 @@ class SolverConfig(Config):
          'epochs with start and end LR being lr/10 and the peak being lr.'))
     multi_stage: List = Field(
         [], description=('List of epoch indices at which to divide LR by 10.'))
+    class_loss_weights: Optional[Union[list, tuple]] = Field(
```

Comment:
If someone sets this field for a task that is not semantic segmentation or chip classification, a validation error should be raised.